third-party/realdds/src/dds-serialization.cpp

```diff
@@ -82,6 +114,39 @@ std::ostream & operator<<( std::ostream & os, RTPSEndpointQos const & qos )
 
 namespace eprosima {
 namespace fastrtps {
+
+
+// Allow j["key"] = qos.lease_duration;
+void to_json( nlohmann::json & j, Duration_t const & duration )
+{
+    if( duration == c_TimeInfinite )
+        j = "infinite";
+    else if( duration == c_TimeInvalid )
+        j = "invalid";
+    else
+        j = realdds::time_to_double( duration );
+}
+
+
+// Allow j.get< eprosima::fastrtps::Duration_t >();
+void from_json( nlohmann::json const & j, Duration_t & duration )
+{
+    if( j.is_string() )
+    {
+        auto & s = rsutils::json::string_ref( j );
+        // TODO switch to rsutils::string::nocaase_equal()
```

Comment:
Just in a comment here but check that the function name is not `caase`

Reply:
Actually, I thing nocase.h is now in... I'll change it